src/DictVectors/initiators.jl

```diff
@@ -42,302 +78,147 @@ Concrete implementations:
 * [`Initiator`](@ref)
 * [`SimpleInitiator`](@ref)
 * [`CoherentInitiator`](@ref)
+* [`NonInitiator`](@ref)
```

Comment:
In the code it is called `NoInitiator`, but maybe `NonInitiator` makes more sense in combination with `NonInitiatorValue`.